misc/book/converter_with_figs.py

```diff
@@ -0,0 +1,63 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Fri Sep  28 20:01:12 2018
+
+@author: aleksandra
+"""
+import os
+
+
+def convert(file_name):
+    '''
+    Convert python script to Markdown
+    '''
+    with open(file_name, 'r', encoding='utf-8') as my_file:
+        my_code = my_file.readlines()
+
+    if 'import matplotlib.pyplot as plt\n' in my_code:

```

Comment:
This is not a very safe method of Python code running. Please, see `runpy`, `importlib`, `code` and `parser` standard library modules.